### DISCUSSION TITLE
Installation Failure

### DISCUSSION THREAD

**Discussion by sushantshah-dev** [nextcloud/all-in-one#4695] [Questions] [answered]
### Steps to reproduce
1. Running the install command given below. 
2. Running the installer (regardless of optional containers).

Though it looks like no one has had the exact same issue as me before as none of the online solutions work for me.

### Expected behavior 
All containers should get installed

### Actual behavior 
I see this error:

NOTICE: PHP message: Slim Application Error
Type: Exception
Code: 0
Message: Could not create container nextcloud-aio-nextcloud: Client error: `POST http://localhost/v1.41/containers/create?name=nextcloud-aio-nextcloud` resulted in a `404 Not Found` response:
{&quot;message&quot;:&quot;No such image: nextcloud/aio-nextcloud:latest&quot;}

### Host OS 
Ubuntu

#### Nextcloud AIO version 
latest

#### Other valuable info 
The server is behind a cloudflare tunnel. ISPs without a CGNAT are difficult to find.

The command I used:
sudo docker run --init --sig-proxy=false --name nextcloud-aio-mastercontainer --restart always --publish 9080:8080 --env APACHE_PORT=11000 --env APACHE_IP_BINDING=0.0.0.0 --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config --volume /var/run/docker.sock:/var/run/docker.sock:ro --env SKIP_DOMAIN_VALIDATION=true --env NEXTCLOUD_DATADIR="/media/NAS-HDD-1/NextCloud" nextcloud/all-in-one:latest

I have erased and retried multiple times, with multiple data directories. The directory doesn't affect the error. I am using alternate ports as they have been taken up by nginx.

![Screenshot 2024-05-23 061616](https://github.com/nextcloud/all-in-one/assets/103421781/49115e43-6007-4141-bbfb-f53e51ac2da4)

**Comment by sushantshah-dev** [comment]
Trying to open the container logs shows:
Slim Application Error
A website error has occurred. Sorry for the temporary inconvenience.

[Go Back](https://192.168.29.5:9080/api/docker/logs?id=nextcloud-aio-apache#)

**Reply by sushantshah-dev** [reply]
Anyone?

**Comment by szaimen** [comment]
Hi, can you run `sudo docker exec -it nextcloud-aio-mastercontainer docker pull nextcloud/aio-nextcloud:latest` and post the output here?

**Reply by sushantshah-dev** [reply]
sushant@server:~$ sudo docker exec -it nextcloud-aio-mastercontainer docker pull nextcloud/aio-nextcloud:latest
[sudo] password for sushant:

latest: Pulling from nextcloud/aio-nextcloud
4abcf2066143: Already exists
4b99432ace8a: Already exists
d3a73aae1c86: Already exists
9294d9185d89: Already exists
0150bd92669e: Pull complete
036a86a4ae38: Pull complete
75d2a8eea798: Pull complete
23484683932f: Pull complete
b8f832e65336: Pull complete
b483e4a592eb: Pull complete
3114680a2e00: Pull complete
e5bd1f252176: Pull complete
bd9046d2310a: Pull complete
08661b04ceba: Pull complete
dfd067a16a2a: Pull complete
Digest: sha256:22d82d1fec35003c7694a75310d634f986e7b28b16a7f6cd65781a322658e1e5
Status: Downloaded newer image for nextcloud/aio-nextcloud:latest
docker.io/nextcloud/aio-nextcloud:latest

sushant@server:~$ sudo docker ps
CONTAINER ID   IMAGE                             COMMAND       CREATED      STATUS                  PORTS
                                                    NAMES
08e6ecf7a461   nextcloud/aio-redis:latest        "/start.sh"   3 days ago   Up 19 hours (healthy)   6379/tcp
                                                    nextcloud-aio-redis
d1f25a94cef7   nextcloud/aio-postgresql:latest   "/start.sh"   3 days ago   Up 19 hours (healthy)   5432/tcp
                                                    nextcloud-aio-database
d4182f39e33a   nextcloud/all-in-one:latest       "/start.sh"   3 days ago   Up 19 hours (healthy)   80/tcp, 8443/tcp, 9000/tcp, 0.0.0.0:9080->8080/tcp, :::9080->8080/tcp   nextcloud-aio-mastercontainer

**Reply by sushantshah-dev** [reply]
I just checked... It's stuck at starting now.

**Reply by sushantshah-dev** [reply]
Logs:

Database:
Setting postgres values...
chmod: /var/run/postgresql: Operation not permitted

PostgreSQL Database directory appears to contain a database; Skipping initialization

2024-05-26 16:10:19.916 IST [14] LOG:  starting PostgreSQL 15.6 on x86_64-pc-linux-musl, compiled by gcc (Alpine 13.2.1_git20231014) 13.2.1 20231014, 64-bit
2024-05-26 16:10:19.916 IST [14] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2024-05-26 16:10:19.916 IST [14] LOG:  listening on IPv6 address "::", port 5432
2024-05-26 16:10:19.935 IST [14] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2024-05-26 16:10:19.986 IST [24] LOG:  database system was shut down at 2024-05-26 16:10:11 IST
2024-05-26 16:10:20.014 IST [14] LOG:  database system is ready to accept connections
2024-05-26 16:10:49.879 IST [42] FATAL:  could not write init file
2024-05-26 16:11:19.950 IST [52] FATAL:  could not write init file
2024-05-26 16:11:20.085 IST [53] FATAL:  could not write init file
2024-05-26 16:11:50.033 IST [63] FATAL:  could not write init file
2024-05-26 16:12:20.102 IST [71] FATAL:  could not write init file
2024-05-26 16:12:20.139 IST [72] FATAL:  could not write init file
2024-05-26 16:12:50.177 IST [82] FATAL:  could not write init file
2024-05-26 16:13:20.199 IST [83] FATAL:  could not write init file
2024-05-26 16:13:20.249 IST [91] FATAL:  could not write init file
2024-05-26 16:13:50.326 IST [101] FATAL:  could not write init file
2024-05-26 16:14:20.247 IST [102] FATAL:  could not write init file
2024-05-26 16:14:20.402 IST [110] FATAL:  could not write init file
2024-05-26 16:14:50.477 IST [121] FATAL:  could not write init file


Nextcloud:
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
Enabling Imagick...
WARNING: opening from cache https://dl-cdn.alpinelinux.org/alpine/v3.19/main: No such file or directory
WARNING: opening from cache https://dl-cdn.alpinelinux.org/alpine/v3.19/community: No such file or directory
Initializing nextcloud 28.0.5.1 ...
rsync: [receiver] write failed on "/var/www/html/apps/bruteforcesettings/js/bruteforcesettings-main.js.map": No space left on device (28)
rsync error: error in file IO (code 11) at receiver.c(380) [receiver=3.2.7]
rsync: [sender] write error: Broken pipe (32)
Initializing finished
New Nextcloud instance.
Installing with PostgreSQL database
Starting Nextcloud installation...

Warning: require_once(/var/www/html/lib/versioncheck.php): Failed to open stream: No such file or directory in /var/www/html/console.php on line 34

Fatal error: Uncaught Error: Failed opening required '/var/www/html/lib/versioncheck.php' (include_path='.:/usr/local/lib/php') in /var/www/html/console.php:34

ace:
#0 /var/www/html/occ(11): require_once()

}
  thrown in /var/www/html/console.php on line 34
Installation of Nextcloud failed!
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
Connection to nextcloud-aio-database (172.18.0.2) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance

Notify Push:
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9001: Try again
Waiting for Nextcloud to start...

**Reply by sushantshah-dev** [reply]
Also:
```sushant@server:~$ sudo docker ps -a
CONTAINER ID   IMAGE                              COMMAND                  CREATED         STATUS
   PORTS                                                                   NAMES
d9fa93c6ca18   nextcloud/aio-apache:latest        "/start.sh /usr/bin/…"   3 minutes ago   Up 3 minutes (healthy)          80/tcp, 0.0.0.0:11000->11000/tcp                                        nextcloud-aio-apache
274a43f4756a   nextcloud/aio-notify-push:latest   "/start.sh"              3 minutes ago   Up 3 minutes (healthy)                                                                                  nextcloud-aio-notify-push
1d68db21e69b   nextcloud/aio-nextcloud:latest     "/start.sh /usr/bin/…"   4 minutes ago   Restarting (1) 54 seconds ago                                                                           nextcloud-aio-nextcloud
bedaa894f244   nextcloud/aio-redis:latest         "/start.sh"              4 minutes ago   Up 4 minutes (healthy)          6379/tcp                                                                nextcloud-aio-redis
2af799478213   nextcloud/aio-postgresql:latest    "/start.sh"              4 minutes ago   Up 4 minutes (unhealthy)        5432/tcp                                                                nextcloud-aio-database
d4182f39e33a   nextcloud/all-in-one:latest        "/start.sh"              3 days ago      Up 19 hours (healthy)           80/tcp, 8443/tcp, 9000/tcp, 0.0.0.0:9080->8080/tcp, :::9080->8080/tcp   nextcloud-aio-mastercontainer```

**Reply by sushantshah-dev** [reply]
Server error. Please check the mastercontainer logs for details. This page will reload after 10s automatically. Then you can check the mastercontainer logs.

**Reply by sushantshah-dev** [reply]
{"level":"info","ts":1716720690.1869705,"msg":"using provided configuration","config_file":"/Caddyfile","config_adapter":""}
[Sun May 26 10:51:30.192430 2024] [mpm_event:notice] [pid 113:tid 129244838968072] AH00489: Apache/2.4.59 (Unix) OpenSSL/3.1.4 configured -- resuming normal operations
[Sun May 26 10:51:30.192717 2024] [core:notice] [pid 113:tid 129244838968072] AH00094: Command line: 'httpd -D FOREGROUND'
[26-May-2024 10:51:30] NOTICE: fpm is running, pid 119
[26-May-2024 10:51:30] NOTICE: ready to handle connections
{"level":"error","ts":1716720690.2117107,"logger":"tls","msg":"could not clean default/global storage","error":"unable to acquire storage_clean lock: creating lock file: write /mnt/docker-aio-config/caddy/locks/storage_clean.lock: no space left on device"}
NOTICE: PHP message: Slim Application Error
Type: AIO\Data\InvalidSettingConfigurationException
Code: 0
Message: /mnt/docker-aio-config/data/ does not have enough space for writing the config file! Not writing it back!
File: /var/www/docker-aio/php/src/Data/ConfigurationManager.php
Line: 534
Trace: #0 /var/www/docker-aio/php/src/Controller/DockerController.php(188): AIO\Data\ConfigurationManager->WriteConfig(Array)
#1 /var/www/docker-aio/php/vendor/slim/slim/Slim/Handlers/Strategies/RequestResponse.php(38): AIO\Controller\DockerController->StartContainer(Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
#2 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(358): Slim\Handlers\Strategies\RequestResponse->__invoke(Array, Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
#3 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\Routing\Route->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#4 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#5 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(315): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#6 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/RouteRunner.php(68): Slim\Routing\Route->run(Object(GuzzleHttp\Psr7\ServerRequest))
#7 /var/www/docker-aio/php/vendor/slim/csrf/src/Guard.php(482): Slim\Routing\RouteRunner->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#8 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(168): Slim\Csrf\Guard->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Slim\Routing\RouteRunner))
#9 /var/www/docker-aio/php/vendor/slim/twig-view/src/TwigMiddleware.php(115): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#10 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Views\TwigMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#11 /var/www/docker-aio/php/src/Middleware/AuthMiddleware.php(38): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#12 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(269): AIO\Middleware\AuthMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#13 /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(76): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#14 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Middleware\ErrorMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#15 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#16 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(199): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#17 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(183): Slim\App->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#18 /var/www/docker-aio/php/public/index.php(185): Slim\App->run()
#19 {main}
Tips: To display error details in HTTP response set "displayErrorDetails" to true in the ErrorHandler constructor.

**Reply by sushantshah-dev** [reply]
I have about 488 GB of free space. Single partition.

**Reply by sushantshah-dev** [reply]
The database has spun up but all the others are stuck at starting waiting for the database

**Comment by szaimen** [comment]
Hi this looks like a hiccup. Can you follow https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-properly-reset-the-instance and try again?

**Reply by sushantshah-dev** [reply]
Tried a couple of times... I also see that the database container doesn't expose the port. Is that the problem?

**Reply by sushantshah-dev** [reply]
{"level":"info","ts":1716845057.312651,"msg":"using provided configuration","config_file":"/Caddyfile","config_adapter":""}
Error: loading initial config: loading new config: starting caddy administration endpoint: listen tcp: lookup localhost on [::1]:53: read udp [::1]:56875->[::1]:53: read: connection refused
[Mon May 27 21:24:17.321043 2024] [mpm_event:notice] [pid 116:tid 140502366804744] AH00489: Apache/2.4.59 (Unix) OpenSSL/3.1.4 configured -- resuming normal operations
[Mon May 27 21:24:17.321420 2024] [core:notice] [pid 116:tid 140502366804744] AH00094: Command line: 'httpd -D FOREGROUND'
[27-May-2024 21:24:17] NOTICE: fpm is running, pid 122
[27-May-2024 21:24:17] NOTICE: ready to handle connections
{"level":"info","ts":1716845058.3546634,"msg":"using provided configuration","config_file":"/Caddyfile","config_adapter":""}
Error: loading initial config: loading new config: starting caddy administration endpoint: listen tcp: lookup localhost on [::1]:53: read udp [::1]:51701->[::1]:53: read: connection refused
{"level":"info","ts":1716845060.3893678,"msg":"using provided configuration","config_file":"/Caddyfile","config_adapter":""}
Error: loading initial config: loading new config: starting caddy administration endpoint: listen tcp: lookup localhost on [::1]:53: read udp [::1]:49376->[::1]:53: read: connection refused
{"level":"info","ts":1716845063.426646,"msg":"using provided configuration","config_file":"/Caddyfile","config_adapter":""}
Error: loading initial config: loading new config: starting caddy administration endpoint: listen tcp: lookup localhost on [::1]:53: read udp [::1]:52746->[::1]:53: read: connection refused


after trying to setup again

**Reply by sushantshah-dev** [reply]
PS: I am not using caddy. And I am behind cloudflare.

**Comment by szaimen** [comment, marked as answer]
Hi, I fear I cannot reproduce any of the problems you are running into. I suspect a problem with the infrastructure you are using. Feel free to create a new topic at https://help.nextcloud.com in order to get help from the community.
